docs/using-with-node.md

```diff
@@ -1,102 +1,142 @@
 # Using with Node
 
-*Note: These instructions describe setting up the alpha version of govuk_frontend and can/will change*
 
-## Add to package.json
+## 1. Download the Node starter app
 
-From this repo, build a local version of the npm package:
+[Download the Node/Express starter app](https://github.com/alphagov/govuk-frontend-alpha-starter-kit-node)
 
-```bash
-gulp package
-```
 
-This will create a tgz file in `dist/npm/`
+## 2. Add the govuk_frontend_alpha package as a dependency
+
+The package is not available on the NPM registry.
 
-Copy the govuk_frontend_alpha-0.0.1.tgz file to the root of your project.
+You will need to manually include it as a dependency.
 
-Amend your package.json file to include govuk_frontend_alpha as a dependency.
+### Amend the package.json file in the root of the starter app
 
 ```bash
 "dependencies": {
- "govuk_frontend_alpha": "file:govuk_frontend_alpha-0.0.1.tgz"
+  "govuk_frontend_alpha": "https://github.com/alphagov/govuk_frontend_alpha/releases/download/0.0.1-alpha/govuk_frontend_alpha-0.0.1-npm.tgz",
  }
 ```
 
-Install the govuk_frontend_alpha package
+### Install the govuk_frontend_alpha package
 
-If you're using [Yarn](https://yarnpkg.com/) to manage packages, you can install the govuk_frontend_alpha dependency with:
+If you're using [Yarn](https://yarnpkg.com/):
 
 ```bash
-yarn install
+yarn
 ```
 
-Instructions for [installing Yarn are here](https://yarnpkg.com/en/docs/install).
-
-or if you're using [NPM](https://www.npmjs.com/) to manage pagckages:
+If you're using [NPM](https://www.npmjs.com/):
 
 ```bash
 npm install
 ```
 
-Note: In future the package will be published to npm and the steps above can be replaced with
 
-```bash
-yarn add govuk-frontend-alpha
+## 3. Set up the starter app
```

Comment:
This is all geared towards the starter app - should we be coupling the installation instructions to a particular app?

This could be reworded to talk about "your node application" more generally. For example the adding of view paths is bespoke to the starter app. I think it would be better to describe the general task - ensuring that path is in the view load paths - and follow up with an example closer to the starter app, but not talk about the starter app specifically.